src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -37,7 +37,7 @@ public User register(String email, String password) {
     public User login(String login, String password) throws AuthenticationException {
         Optional<User> user = userService.findByEmail(login);
         String encodedPassword = passwordEncoder.encode(password);
```

Comment:
You are no longer using this variable, so you should use it instead of user.get().getPassword() or delete it